pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1492,10 +1493,45 @@ class Build {
                                                     file=$(basename "$f")
                                                     mv "$f" "${dir}/unsigned_${file}"
                                                     curl -o "$f" -F file="@${dir}/unsigned_${file}" -F entitlements="@$ENTITLEMENTS" https://cbi.eclipse.org/macos/codesign/sign
-                                                    chmod --reference="${dir}/unsigned_${file}" "$f"
-                                                    rm -rf "${dir}/unsigned_${file}"
+                                                    TESTMACSIGN=`grep -i "Apple Certification Authority" "$f"|wc -l`
```

Comment:
Is there enough logic here to extract into its own function?

Reply:
What about using flag`--fail`, which will prevent curl from outputting the html file and return error 22.

`curl --fail --silent --show-error -o "$f" -F file="@${dir}/unsigned_${file}" -F entitlements="@$ENTITLEMENTS" https://cbi.eclipse.org/macos/codesign/sign`, similar to https://github.com/adoptium/temurin-build/blob/master/sign.sh#L143

Reply:
I think in this context, a shell being run from within a groovy pipeline, this is probably the least impactful solution, especially as its limited to the one job/platform, ie MacOS signing.